views/views.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/go-chi/jwtauth"
 	"github.com/gorilla/csrf"
 	"github.com/s-gv/orangeforum/models"
+	"github.com/s-gv/orangeforum/orangemiddleware"
```

Comment:
This can be `github.com/s-gv/orangeforum/middleware`

Reply:
done